shell/platform/fuchsia/dart_runner/tests/startup_integration_test/dart_jit_runner/dart-jit-runner-integration-test.cc

```diff
@@ -73,10 +75,15 @@ TEST_F(RealmBuilderTest, DartRunnerStartsUp) {
   // Add Dart server component as a child of Realm Builder
   realm_builder.AddChild(kDartJitEchoServer, kDartJitEchoServerUrl,
                          ChildOptions{.environment = kDartRunnerEnvironment});
+
+  // Route base capabilities to the Dart JIT runner
   realm_builder.AddRoute(
-      Route{.capabilities = {Protocol{"fuchsia.logger.LogSink"}},
+      Route{.capabilities = {Protocol{"fuchsia.logger.LogSink"},
+                             Protocol{"fuchsia.tracing.provider.Registry"},
+                             Protocol{"fuchsia.posix.socket.Provider"},
+                             Protocol{"fuchsia.intl.PropertyProvider"}},
             .source = ParentRef(),
-            .targets = {kDartJitEchoServerRef}});
+            .targets = {kDartJitRunnerRef}});
```

Comment:
You should still also route LogSink to the EchoServer (in addition to routing LogSink to the runner).